tests/common/helpers/platform_api/fan_drawer.py

```diff
@@ -67,3 +67,6 @@ def set_status_led(conn, index, color):
 
 def get_status_led(conn, index):
     return fan_drawer_api(conn, index, 'get_status_led')
+
```

Comment:
Per [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be two blank lines between top-level function definitions.